### ISSUE TITLE
Page threw exception during rendering and feature request

### ISSUE THREAD

**Issue by kest874** [radzenhq/radzen-blazor-studio#31]
```
C:\Dev\Audits\server\Pages\EnvironmentalAuditViewEdit.razor(20437,59): error CS1003: Syntax error, ',' expected
C:\Dev\Audits\server\Pages\EnvironmentalAuditViewEdit.razor(20437,95): error CS1001: Identifier expected
C:\Dev\Audits\server\Pages\EnvironmentalAuditViewEdit.razor(20437,96): error CS1003: Syntax error, ',' expected
C:\Dev\Audits\server\Pages\EnvironmentalAuditViewEdit.razor(20437,98): error CS1002: ; expected
C:\Dev\Audits\server\Pages\EnvironmentalAuditViewEdit.razor(20524,72): error CS1002: ; expected

   at Radzen.Server.ProjectContext.Compile(Compilation compilation, AssemblyLoadContext context) in /Users/korchev/github/radzen-next/Radzen.Server/ProjectContext.cs:line 122
   at Radzen.Server.ProjectServer.Compile(String fileName, String source, RazorProjectEngine projectEngine) in /Users/korchev/github/radzen-next/Radzen.Server/ProjectServer.cs:line 776
   at Radzen.Server.ProjectServer.CompilePage(String fileName, String source, RazorProjectEngine engine) in /Users/korchev/github/radzen-next/Radzen.Server/ProjectServer.cs:line 318
   at Radzen.Server.ProjectServer.Render(String fileName, String source) in /Users/korchev/github/radzen-next/Radzen.Server/ProjectServer.cs:line 392
   at Radzen.Server.ProgramController.Render(RenderRequest request) in /Users/korchev/github/radzen-next/Radzen.Server/ProgramController.cs:line 325
```

Above error on this:

`<RadzenLabel Component="ProperSelectionUseRating" style="color: #212529; font-weight: bold; width: 100%" Text="@($"{("2: " + getEnvironmentalQuestionsResult.Where(r => r.ID == 2).FirstOrDefault()?.Question)}")">`

This is a project that was developed and renders fine in Radzen.

Also one more question in the source viewer  is it possible to add a "go to offset" would be handy to jump right to offset 20437 to determine where the error is being generated. (I used Notepad+ with go to offset)

**Comment by kest874** [comment]
Yes no problem it  does compile and run.

**Comment by kest874** [comment]
@akorchev Project sent.

**Comment by akorchev** [comment]
> Also one more question in the source viewer is it possible to add a "go to offset" would be handy to jump right to offset 20437

That code and offset does not exist in your file (it is design time code generated by Radzen). Go to offset cannot work.

**Comment by akorchev** [comment]
We released 1.0.0.preview.12 which should address the issue.
